src/kitchen/components/button/Button.stories.tsx

```diff
@@ -11,7 +11,7 @@ const defaultFormat = {
 };
 
 export default {
-	title: 'Editorial/src-ed-button/Button',
+	title: 'Kitchen/source-kitchen-react/Button',
```

Comment:
Does this component now need to be called something else (e.g. `EditorialButton`) as we can't use the package name to differentiate it from the standard Source `Button`?

Reply:
@guardian/dotcom-platform what do you think? 

- `EditorialButton`
- `PillarButton`
- Something else?

Reply:
Adding an option: `ThemeButton`?

Reply:
Thanks @oliverlloyd I personally think the word `Theme` is already used more generally to support, for example, the various Reader Revenue themes and the brand themes (see [Figma](https://www.figma.com/file/fq2gb2YycRKD3FwojpyOPe/%E2%9C%AD-Buttons?node-id=1504%3A1626)). Do you see Theme as something specific to Format?

Reply:
`Format.Theme` is a superset of `Pillar` which also includes `Labs` and `SpecialReport`

https://github.com/guardian/libs/blob/main/src/format.ts#L16

Reply:
Okay, thanks. I think we still need a more specific qualifier so as not to confuse people who aren't familiar with Format. Maybe `EditorialThemeButton`?

Reply:
I think in that case, maybe `EditorialButton` makes more sense? This seems to contain all the meaning needed and then you don't have the confusing aspect of 'theme' (which, I agree, is a bit overloaded)

Reply:
In the migration to `@guardian/libs`, `Format` becomes [`ArticleFormat`](https://github.com/guardian/libs/blob/main/src/format.ts). In fact everything related to `ArticleFormat` gets an `Article` prefix. Maybe we should reflect that in Source too, with `ArticleButton`?

Reply:
Wasn't it that `ArticleFormat` is named such because it specifically relates to articles? With the thinking being you would later have a `ContainerFormat` (or `SliceFormat`, `SectionFormat`?) which represents Fronts. `SectionFormat` would contain and array of articles which each have `ArticleFormat`.

I mention this because I don't think the same model exists for a button. They can be used everywhere in the editorial space and have a much simpler design pattern, one not limited to articles.

Reply:
On the other hand, if we follow this thinking through, you'd have 

```ts
<EditorialButton format={myFormatObject} />
```

where `myFormatObject` would be typed as `ArticleFormat`.  So if you can only ever pass in `ArticleFormat` then maybe `ArticleButton` works?

🤷

Reply:
Good thoughts @oliverlloyd. I guess the `format` prop could expect something that's either an `ArticleFormat` or `ContainerFormat` But I think you're right in that `<EditorialButton>` better reflects the context in which this button can be used.

Reply:
I've [renamed to EditorialButton](https://github.com/guardian/source/pull/963/commits/c3391a09ad74131043e941435a939d77193e8083) unless anyone has any last minute alternatives